autofit/mapper/prior_model/abstract.py

```diff
@@ -411,6 +411,37 @@ def instance_from_prior_medians(self):
             unit_vector=[0.5] * len(self.prior_tuples)
         )
 
+    def log_priors_from_vector(
+            self,
+            vector,
+    ):
+        """
+        Compute the log priors of every parameter in a vector, using the Prior of every parameter.
+
+        The log prior values are used by Emcee to map the log likelihood to the poserior of the model.
+
+        Parameters
+        ----------
+        vector: [float]
```

Comment:
Prefer putting type annotations in the signature.

You can use the typing library for collections etc.

For example:
vector: List[float]

Reply:
I agree - but for now I'll leave it in the docstring (I have added into the function inputs). I need to check whether Sphinx can use the function inputs when auto generating docs, I know that it used to use the numpy docstrings to do this.